### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by yurg** [elmsln/elmsln#2137]
Bug: After several attempts to install it via install script found at https://elmsln.readthedocs.io/en/latest/INSTALL/ can't get it to load people.domain.tld / online.domain.tld. 
Have been able to reach site(s) for a short time only after drush \@network up yet after loading admin pages and walking through site it gets unresponsive with bunch of errors thrown in apache error.log / syslog / mysql error.log;  complaining regarding php timeout exceeded  / various Drupal modules  errors, some mysql resource issues , and so on and so forth, hard to tell it's a single source.    
Server on Ubuntu 16.4 with 4Gb of RAM. Have tried nginx & php7-fpm (which doesn't seems to work entirely), yet have had to downgrade to Apache & php 5.6-cli to reach at least a starting page of the application.  Will you please advice about where to start debugging?   Mean, I would like to help, being a Drupal developer for more than a decade I've seen a lot, yet elmsln has been able to surprise me. 
Thanks in advance!

### ELMSLN info: ### 

Info | ELMSLN commit | 402692e8032b1ac715142065501af298a084a4a4
-- | -- | --
Use this to help reporting issues to the community. Here's a link to the exact commit which can be useful for referencing.
Info | ELMSLN branch | master
Here's a link to the branch in our repo which can be useful for referencing.
OK | ELMSLN connection status | Connection keychain detected!
The following tools can be accessed: assessment cis cle cpr discuss ecd editorial elmsmedia icor inbox lq mooc ulmus
OK | ELMSLN version | 0.8.0
Code and configuration match.

**Comment by btopro** [comment]
Hi @yurg thanks for filing this issue. https://www.elmsln.org/install-guide has the recommended way of filling out the criteria to generate a 1-line install. It sounds like maybe you went through the install manually (which isn't recommended at this point and isn't maintained). If that's not the case let me know.

Let me know more info from there and we can help. We run elms:ln on Cent 6.x/php 5.6 and Ubuntu 16.x php 7.x in production environments, multiple, and Ubuntu 16 + PHP 7 is our default.

**Comment by yurg** [comment]
Hey @btopro, thank you for prompt reply;  exactly, it seems I should have checked ubuntu16-install.sh at early stages, right after realising my shiny new fresh latest nginx&php&mysql setup doesn't fit internal logic of elmsln installer. So this issue is mostly a way to show my confusion and angriness, sorry for that! Any chance to put missing pieces together apart  from wiping out everything and start from bare Ubuntu? Thanks!

**Comment by btopro** [comment]
I mean you could try it and see but we're best able to debug from a clean box

**Comment by yurg** [comment]
I think it worth reopening. Couple of suggestions regarding https://www.elmsln.org/install-guide  if I may.

1. "Organisation" label: shouldn't it be "Site URL"? I've used a real organisation title and have had to purge the whole setup, because the first word  of organisation name has been used in all links. 

2. Whatever puts at "Share anonymous statistics.. " it always stays "yes".

3.   Since installer is strict regarding all things and inevitable conflicts with anything pre-installed, maybe it makes sense to 
a) add something like "[sudo] apt-get purge apache* mysql* php* && apt-get autoremove &&  apt-get autoclean &&  apt-get dist-upgrade" at the very beginning of the script OR
b) Add a noticeable warning / info  somewhere at the beginning of https://www.elmsln.org/install-guide page, which would say something like "You have to set this up only on fresh Ubuntu without any Apache\Php\Mysql pre-installed". I've learnt this a hard way so think such notice will save a lot of time for anyone else. 

4. May be it worth excluding HTTPS option from the installer since it doesn't seems to work properly even on fresh Ubuntu. There is something in letsencrypt part which prevents it from working. 

5. I am not really sure if installer keeps it's own log, yet it would be useful: I've mentioned some errors regarding Composer & Drush and may be other parts, which, probably, should be reported back here. Would be nice to have something like "All issues have been saved to /var/www/elmsln/install.log, please report it back to github.com/elmsln/elmsln/issues/ if there is any installation issues" at the end of the setup process. 

Sorry if it sounds like too much criticism, I'm just trying to share experience gathered for several days spent on set this up. 
Thanks!

**Comment by btopro** [comment]
nope, good to get feedback, dono when we can get to it though

**Comment by yurg** [comment]
Done: https://github.com/elmsln/elmsln/issues/2141 
Thank you!
